app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ReservationsController < ApplicationController
+  def new
+    render :new, locals: { reservation: Reservation.new, seance: Seance.find(params[:seance_id]) }
+  end
+
+  def create
+    reservation = CreateReservation.new(params:)
```

Comment:
You are passing `params` as named parameter, I was not sure about this syntax, but you attach the video and proved that it work. 🚀 
```suggestion
    reservation = CreateReservation.new(params: params)
```

You can consider passing explicit arguments to `CreateReservation`. Benefits I see in this approach:
- you can see from controller what exact arguments reservation needs to be created
- you can reuse the `CreateReservation` service in some different use case (not everywhere you have access to this specific `params`).
In situation when creating reservation needs a lot of arguments you can pass it as one `params` (or any other compact struct).
This is just my suggestion, your solution is also correct, you can chose what you want.

```suggestion
    reservation = CreateReservation.new(email: email, seance_id: seance_id seats: seats)
```